discovery-provider/src/app.py

```diff
@@ -475,7 +475,7 @@ def configure_celery(celery, test_config=None):
             },
             "index_related_artists": {
                 "task": "index_related_artists",
-                "schedule": timedelta(seconds=60),
+                "schedule": timedelta(seconds=300),  # todo: make every few days
```

Comment:
what barometer can we use to determine how often we are allowed to refresh this? would be nice to do more freq than a few days